platform/posix/utest/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+project ("posix unit test")
```

Comment:
Should this be "Reconnect unit test" specifically as POSIX is an umbrella term for all things platform related

Reply:
Agreed, but this file will also be reused for POSIX clock unit tests.